corehq/apps/geospatial/forms.py

```diff
@@ -108,7 +109,16 @@ class Meta:
         # ),
         choices=DISBURSEMENT_ALGORITHM_OPTIONS,
         required=True,
-        help_text=_("The algorithm which will be used to disburse cases between users"),
+        help_text=format_html_lazy(_('''
+            <span data-bind="visible: selectedAlgorithm() == '{}'">
```

Comment:
nit: I think people do tend to mess up the syntax with translations. So it's safer to limit to plain text.
We can't leave the span bit out of translation here, like, have two separate translations for text?
```
<span data-bind="visible: selectedAlgorithm() == f'{GeoConfig.RADIAL_ALGORITHM}'">
 _("Uses the straight-line distance between users and cases to...")
</span>
<span data-bind="visible: selectedAlgorithm() == f'{GeoConfig.ROAD_NETWORK_ALGORITHM}'">
_("Takes distance along roads between ...")
</span>
```

Reply:
That's fair. I've refactored this a little so that only the plain text is being translated ([e3bc902](https://github.com/dimagi/commcare-hq/pull/34792/commits/e3bc90272571002298967acc3a2c3b82ce832534)).

Reply:
looks great, thank you.